components/declineButton.tsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable react/button-has-type */
+const DeclineButton: React.FunctionComponent = () => (
+  // eslint-disable-next-line react/jsx-no-comment-textnodes
+  <div className="flex space-x-4">
+    <div>
+      <button className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">
+        Decline
+      </button>{" "}
```

Comment:
Extra space here not needed:
```suggestion
      </button>
```